master/master.cfg

```
@@ -288,6 +288,24 @@ for branch_num, (git_url, branchname, git_branch) in enumerate(git_branches):
 stable_pull_request_builders = []
 all_pull_request_builders = []
 
+# Avoid a build to be started between start and end time and delay such build
+# at end time
+def no_builds_between(start, end):
+    def f(builder, requests):
+        now = datetime.now()
+        if start <= now.hour < end:
+            # Calculate the delay until end time
+            delay = ((end - now.hour) % 24) * 3600 - now.minute * 60 - now.second
+            builder.master.reactor.callLater(
+                delay,
+                builder.buildset_manager.submitBuildSet,
+                requests[0]
```

```suggestion
                requests[0],
```

Just for consisetncy across the file.